watch_few.go

```diff
@@ -1,17 +1,15 @@
 //go:generate sh -c "go run watch-gen/main.go >watch_few.go"
```

Comment:
The change here looks good but you actually edit watch-gen/main.go and then `go generate` to make this file and then check it in.